actual/docker-compose.yml

```
@@ -0,0 +1,15 @@
+version: '3.7'
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: actual_web_1
+      APP_PORT: 5006
+      PROXY_AUTH_ADD: "false"
+
+  web:
+    user: 1000:1000
+    image: ghcr.io/actualbudget/actual:sha-4dba958@sha256:a22cafc52d51d7f84b1f1d30a3fbd6ad53b792edc04f627067216e9f0517d714
```

Use the tag for the version that is being installed.
```suggestion
    image: ghcr.io/actualbudget/actual:25.5.0@sha256:3b486fbc0db02839917f65ed277aa509980f710b6e96dde966f5219983978179
```